tests/integration-tests/tests/scaling/test_scaling/test_nodewatcher_terminates_failing_node/pcluster.config.yaml

```diff
@@ -1,22 +0,0 @@
-Image:
```

Comment:
have you removed the test code as well?

Reply:
It was already removed by @hanwen-pcluste that missed to remove these files.